src/test/regress/bin/normalize.sed

```diff
@@ -161,6 +161,9 @@ s/Citus.*currently supports/Citus currently supports/g
 s/prepared transaction with identifier .* does not exist/prepared transaction with identifier "citus_x_yyyyyy_zzz_w" does not exist/g
 s/failed to roll back prepared transaction '.*'/failed to roll back prepared transaction 'citus_x_yyyyyy_zzz_w'/g
 
+# Table alias'es in explain outputs
+s/(->\ +Seq Scan on\ +)(.*)_([0-9]+)_([0-9]+) \2_([0-9]+)/\1\2_\3_\4 \2_xxx/g
```

Comment:
Can you write an example of what this is converted to for readability above this

Also why are we only interested in `Seq Scan`?

Reply:
>Also why are we only interested in Seq Scan?
Because we want to normalize aliases in seq scan logs

Reply:
>Because we want to normalize aliases in seq scan logs

My question is mostly around why this only happens with `Seq Scan`, for example does it not happen with `Bitmap Heap Scan`?

As in could the first part be:

```
(.*Scan on\ +)(.*)(_[0-9]+)(_[0-9]+) \2(_[0-9]+|_xxx)?
```

>Actually we can already see the examples by looking into the commit, is it needed ?

I find this more of `useful` than `needed` when writing regex, because if there is an example right above it, then it is much easier to understand what it is doing when looking at it. Sure you can go to the commit to see what it is doing but it adds more steps.

Something like:

```
... Scan on <table_name>_<id>_<id2> <table_name>_<id3>
->
... Scan on <table_name>_<id>_<id2> <table_name>_xxx
```

Reply:
https://github.com/citusdata/citus/pull/4295#discussion_r517931421

now all makes sense, I will address those